`test_denyoom_commands[df_factory0]` parameters rejected (weak test)

```
E       AssertionError: Weak testcase: too little used memory
E       assert 262492296 > 268435456
```

https://github.com/dragonflydb/dragonfly/actions/runs/25368737302/job/74386346444#step:7:1246

example failure trace

```
dragonfly/generic_test.py::test_denyoom_commands[df_factory0] 
-------------------------------- live log call ---------------------------------
[2026-05-29 11:15:20.428 INFO] Starting /__w/dragonfly/dragonfly/build/dragonfly --proactor_threads=1 --maxmemory=256mb --oom_deny_commands=get --dbfilename= --noversion_check --vmodule=dragonfly_connection=1,db_slice=1,listener_interface=1,main_service=1,rdb_save=1,rdb_load=1,replica=1,cluster_family=1,engine_shard=1,dflycmd=1,snapshot=1,streamer=1 --jsonpathv2 --replica_delete_expired=false --omit_basic_usage --latency_tracking --log_dir=/tmp/dragonfly_logs/test_denyoom_commands_df_factory0_ --serialization_max_chunk_size=300000 --serialize_hnsw_index=true --deserialize_hnsw_index=true --fiber_safety_margin=4096 --force_epoll=true --port=-1, pid 19922
[2026-05-29 11:15:20.501 INFO] Process 19922 started after 0.06 seconds. port=39509
Used memory 262492296, rss 338268160
```

```
______________________ test_denyoom_commands[df_factory0] ______________________

df_factory = Factory({})

    @pytest.mark.asyncio
    async def test_denyoom_commands(df_factory):
        df_server = df_factory.create(proactor_threads=1, maxmemory="256mb", oom_deny_commands="get")
        df_server.start()
        client = df_server.client()
        await client.execute_command("DEBUG POPULATE 7000 size 44000")
    
        min_deny = 256 * 1024 * 1024  # 256mb
        info = await client.info("memory")
        print(f'Used memory {info["used_memory"]}, rss {info["used_memory_rss"]}')
>       assert info["used_memory"] > min_deny, "Weak testcase: too little used memory"
E       AssertionError: Weak testcase: too little used memory
E       assert 262492296 > 268435456
```

so https://github.com/dragonflydb/dragonfly/actions/runs/26632875998/job/78485816824 does not fail in any unique way different from this issue. its the same error

--- a/tests/dragonfly/generic_test.py
+++ b/tests/dragonfly/generic_test.py
@@ -232,7 +232,9 @@ async def test_reply_guard_oom(df_factory, df_seeder_factory):
 
 @pytest.mark.asyncio
 async def test_denyoom_commands(df_factory):
-    df_server = df_factory.create(proactor_threads=1, maxmemory="256mb", oom_deny_commands="get")
+    df_server = df_factory.create(
+        proactor_threads=1, maxmemory="256mb", oom_deny_commands="get", rss_oom_deny_ratio=-1
+    )
     df_server.start()
     client = df_server.client()
     await client.execute_command("DEBUG POPULATE 7000 size 44000")

